app/javascript/react/containers/ParkFormEditContainer.js

```diff
@@ -89,8 +89,10 @@ class ParkFormEditContainer extends Component {
         })
         .then(response => response.json())
         .then(body => {
-            this.setState({ updatedPark: body.park, error: body.errors })
-            browserHistory.push(`/parks/${this.props.params.id}`)
+            this.setState({ park: body.park, error: body.errors })
+            if (body.park != {}) {
```

Comment:
let's be explicit about equality with the `!==` operation.

`body.park !== {}`